backend/config/config.go

```diff
@@ -59,6 +60,7 @@ const (
 // ethCoinConfig holds configurations for ethereum coins.
 type ethCoinConfig struct {
 	DeprecatedActiveERC20Tokens []string `json:"activeERC20Tokens"`
+	BlockExplorerTxPrefix       string   `json:"blockExplorerTxPrefix"`
```

Comment:
i wonder if it's maybe better to add a separate field to the backend config `blockExplorers`, so it's all concentrated in one location :thinking:

Reply:
that makes sense.